backend/internal/controller/server/msg_handler.go

```diff
@@ -37,7 +33,7 @@ func (s server) TableMessagesHandler(w http.ResponseWriter, r *http.Request) {
 		response, err := s.createResponse(receivedMsg)
 
 		if err != nil {
-			log.Error(err)
+			log.Debug(err)
```

Comment:
Why is it log.Debug, shouldn't be Error as it was?

Reply:
It is "unrecognized message type" error. For now we don't handle incoming ball position messages so it would spam this error in logs.

Reply:
If an error repeatedly occurs, spamming it in logs is expected, isn't it?

Reply:
It is, changed it back to log.Error